src/pages/manage_dac/ManageEditDac.jsx

```diff
@@ -0,0 +1,385 @@
+import * as ld from 'lodash';
+import React, { useEffect, useState } from 'react';
+import AsyncSelect from 'react-select/async';
+import { DAC } from '../../libs/ajax/DAC';
+import { Models } from '../../libs/models';
+import { PromiseSerial } from '../../libs/utils';
+import { Alert } from '../../components/Alert';
+import { Link } from 'react-router-dom';
+import { DacUsers } from './DacUsers';
+import { Notifications } from '../../libs/utils';
+import editDACIcon from '../../images/dac_icon.svg';
+import backArrowIcon from '../../images/back_arrow.svg';
+import { Spinner } from '../../components/Spinner';
+import { Styles } from '../../libs/theme';
+
+export const CHAIR = 'chair';
+export const MEMBER = 'member';
+const CHAIRPERSON = 'Chairperson';
+
+export default function ManageEditDac(props) {
+  const [state, setState] = useState({
+    error: Models.error,
+    dirtyFlag: false,
+    dac: Models.dac,
+    chairsSelectedOptions: [],
+    chairIdsToAdd: [],
+    chairIdsToRemove: [],
+    membersSelectedOptions: [],
+    memberIdsToAdd: [],
+    memberIdsToRemove: [],
+    searchInputChanged: false
+  });
+  const [isLoading, setIsLoading] = useState(true);
+  const dacId = props.match.params.dacId;
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const fetchedDac = await DAC.get(dacId);
+        setState(prev => ({ ...prev, dac: fetchedDac }));
+      }
+      catch(e) {
+        Notifications.showError({text: 'Error: Unable to retrieve current DAC from server'});
+      }
+    };
+    fetchData();
+    setIsLoading(false);
+  }, [dacId, setState]);
+
+  const okHandler = async (event) => {
+    event.preventDefault();
+
+    let currentDac = state.dac;
+    if (state.dirtyFlag) {
+      // TODO: removed the admin check .. not sure how to get the user role if not passed in as prop
```

Comment:
Is this TODO actionable or meant to be ticketed, or is it serving as a FYI?

Reply:
It is actionable!! I missed that. Thanks for the call out 😄